packages/instrumentation-openai/src/types.ts

```diff
@@ -1,3 +1,14 @@
 import { InstrumentationConfig } from "@opentelemetry/instrumentation";
 
 export interface OpenAIInstrumentationConfig extends InstrumentationConfig {}
+
+export interface TraceloopManagedPromptAttributeObject {
+  _traceloopManagedPromptAttributes?: TraceloopManagedPromptAttributes;
+}
+export interface TraceloopManagedPromptAttributes {
```

Comment:
I dont like that this is here, but on the other hand I didnt want to add the traceloop sdk as a dependency...

thoughts on where to put this? (semantic conventions maybe?)